### DISCUSSION TITLE
Social preview

### DISCUSSION THREAD

**Discussion by completely-burned** [completely-burned/glowing-octo-shame-arma2#86] [Show and tell]
Сюда можно выкладывать ссылки на видео и скриншоты с этой миссией.  
Хочу поставить на Social preview.

Чтобы картинка из другого источника была видна здесь нужно такое например
```
[![имя ссылки](https://img.youtube.com/vi/lNjQkyvkaas/hqdefault.jpg)](https://youtu.be/lNjQkyvkaas)
```
[![lNjQkyvkaas](https://img.youtube.com/vi/lNjQkyvkaas/hqdefault.jpg)](https://youtu.be/lNjQkyvkaas)

**Comment by completely-burned** [comment]
[![chat](https://steamuserimages-a.akamaihd.net/ugc/1813269944404755817/8816BB6E6900E22CFA551617B659505774A0B22D/)](https://s.team/chat/MfOtOM0G)

[![](https://steamuserimages-a.akamaihd.net/ugc/1813269944404759180/A1CC82017CEB5E9A189B5BBC6FCA3D91639623BA/)](https://s.team/chat/MfOtOM0G)

**Comment by black-okean** [comment]
https://vk.ru/clip-226096831_456268626?c=1
